modules/cnf-running-the-performance-creator-profile-offline.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+[id="running-the-performance-creator-profile-wrapper_{context}"]
+= Running the Performance Profile wrapper script
+
+The performance profile wrapper script simplifies the running of the Performance Profile Creator (PPC) tool. It hides the complexities associated with running `podman` and specifying the mapping directories
+and it enables the creation of the performance profile.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* A node with `podman` and OpenShift CLI (`oc`) installed.
+* Access to `must-gather` data.
+
+[NOTE]
+====
+The user is expected to run `must-gather` and compress the output. For more information see the section Gathering data about your cluster under Support
+====
+
+.Procedure
+
+. Create a file on your local machine named, for example, `run-perf-profile-creator.sh`:
++
+[source,terminal]
+----
+$ vi run-perf-profile-creator.sh
+----
+
+. Paste the following code into the file:
++
+[source,bash]
+----
+#!/bin/bash
```

Comment:
Please ensure you have the latest version :) , I didn't check

Reply:
@marcel-apf I executed this and it worked OK. How do we ensure we keep this up-to-date if changes are made to this. I notice you did a wget to retrieve the latest is that something we want to advertise in the docs ? I can check close to 4.8 release date for the latest and greatest and manually paste it in the docs here. If we are happy to go with that.

Reply:
Yes, please do (manual copy-paste the script), we cannot "wget" from the upstream github repository. We will update the doc when necessary. Thank you!

Reply:
@gsr-shanks @marcel-apf To be honest I don't think we have any other solution to this. I take your point but I think this is more up to dev where to host this script and something outside documentation.